docs/diagrams/UiClassDiagram.puml

```diff
@@ -34,15 +36,18 @@ MainWindow *-down->  "1" CommandBox
 MainWindow *-down-> "1" ResultDisplay
 MainWindow *-down-> "1" StudentListPanel
 MainWindow *-down-> "1" StatusBarFooter
+MainWindow *-down-> "1" LessonListPanel
 MainWindow --> "0..1" HelpWindow
 
 StudentListPanel -down-> "*" StudentCard
+LessonListPanel -down-> "*" LessonCard
```

Comment:
Yep, this seems to be the correct multiplicity from lesson list to lessons. Should we call this "Calendar" to better represent what we actually  call it?